examples/teleport-usage/teleport_usage.py

```diff
@@ -0,0 +1,110 @@
+"""
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import datetime
+import boto3
+import os
+
+from boto3.dynamodb.conditions import Key, Attr
+
+
+# Check for AWS DynamoDB table name and region ENV, if none prompt
+table_name = os.getenv("TABLE_NAME")
+if not table_name:
+    table_name = input("DynamoDB events table name: ")
+
+aws_region = os.getenv("AWS_REGION")
+if not aws_region:
+    aws_region = input("AWS Region for DynamoDB: ")
+
+# Check ENV for start time, end time is always 30 days later.
+start_date = os.getenv("START_DATE")
+if not start_date:
+    now = datetime.datetime.now()
+    start_date = now - datetime.timedelta(days=30)
+else:
+    start_date = datetime.datetime.strptime(start_date,'%Y-%m-%d')
+
+end_date = start_date + datetime.timedelta(days=30)
+
+# Configure DynamoDB for query
+dynamodb = boto3.resource("dynamodb", region_name=aws_region)
+table = dynamodb.Table(table_name)
+
+# Generating data message
+start_date_str = start_date.strftime("%b %d %y")
+end_date_str = end_date.strftime("%b %d %y")
+day_range = (end_date - start_date).days
+d_range = (end_date - start_date).days + 1

```

Comment:
I think you should remove `day_range` (or rename `d_range` to `day_range`) and keep the `(end_date - start_date).days + 1` value.  Below this we print `{day_range} days` but that's not actually accurate, since the number of days is actually `d_range`.